src/main/resources/com/mesosphere/velocity/marathon/MarathonRecorder/help-forceUpdate.html

```diff
@@ -0,0 +1,16 @@
+<div>
+    <p>
+        This parameter decides how to handle updating a Marathon application that has an active deployment.
+    </p>
+    <p>
+        If this is set to <code>true</code> (checked), then a new deployment will be forced and the current deployment
+        canceled. If this is set to <code>false</code> (unchecked), then this plugin will attempt to update the Marathon
+        application every couple of seconds until a timeout is reached. If the active deployment does not complete within
+        the time frame, then an error will be returned and the job marked as failed.
```

Comment:
If `forceUpdate` is not set, then the update to Marathon will fail (non-success status returned from Marathon) if an active deployment is already in progress. In that scenario, the deploy plugin does have retry logic and will retry every X seconds for a total of Y times or until the update succeeds. If `forceUpdate` is set, Marathon will accept the update and send a success status.